znai-java/src/main/java/com/twosigma/znai/java/extensions/JavaIncludePlugin.java

```diff
@@ -110,4 +141,8 @@ private String extractBodyOnly(JavaMethod method) {
 
         return result;
     }
+
+    private Collector<CharSequence, ?, String> collectorWithSeparator() {
+        return Collectors.joining(isSignatureOnly ? "\n" : "\n\n");
```

Comment:
Maybe there should be a different separator for isBodyOnly, such as a horizontal line. 2 line breaks may not make it clear that they're different methods; some people put more than 1 line break in methods for example.